src/util/request-manager.js

```diff
@@ -382,9 +382,7 @@ export default class RequestManager {
         return false;
       }
       params.retryAttempts = attempts + 1;
-      if (typeof params.cleanup === 'function') {
-        params.cleanup();
-      }
+
       opts.retryReason = reason;
```

Comment:
It's unclear whether this was ever used anywhere. I could not find any traces of `request-manager` having a `cleanup` function bound to its options, but I'll do one more pass just make sure.